src/clarity-angular/datagrid/built-in/filters/datagrid-string-filter.spec.ts

```diff
@@ -95,24 +75,33 @@ export default function(): void {
             expect(input.focus).toHaveBeenCalled();
         }));
 
-        xit("closes when the user presses Enter in the input", function() {
+        it("offers two way binding on the filtered state", function () {
+            context.testComponent.filterValue = "M";
+            context.clarityDirective.filterValueChange.subscribe((m: string) => {
+                expect(m).toBe("M");
+            });
+            context.detectChanges();
+            expect(context.clarityDirective.filter.value).toBe("M");
+        });
+
+        xit("closes when the user presses Enter in the input", function () {
             // TODO
             openFilter();
         });
 
-        xit("closes when the user presses Escape in the input", function() {
+        xit("closes when the user presses Escape in the input", function () {
             // TODO
             openFilter();
         });
 
-        xit("exposes an Observable to follow filter changes", fakeAsync(function() {
-            // TODO
+        xit("exposes an Observable to follow filter changes", fakeAsync(function () {
+            // TODO -  Can I prune this (Moved to filters provider line #58?)
```

Comment:
Yes, I think you can remove this. We'll add more tests in the future if we have time.